whitepaper/whitepaper.md

```diff
@@ -1437,6 +1440,47 @@ provided here for ease of understanding:
 
 <img width="700" src="./plugin-consortium-manual-bootstrap-sequence-diagram.png">
 
+## 5.7 Deployment Scenarios
+
+There's a near-infinite possible deployment scenarios which is intended to be one of the benefits of using Cactus where
+you are not constrained by strong opinions imposed upon you by the framework but instead can tailor it to your needs
+while still getting value out of it.
```

Comment:
those statements are meaningless, and I suggest to remove.

Reply:
> those statements are meaningless, and I suggest to remove.

@sfuji822 Thank you very much! Done: 4cf8081cb77528abeedbc6bcc2e4ab54fe927b80